repo/packages/J/jenkins/10/config.json

```diff
@@ -63,6 +63,22 @@
                 }
             }
         },
+        "security": {
+            "description": "Security configuration properties for Jenkins on DC/OS.",
+            "type": "object",
+            "properties": {
+                "strict-mode": {
+                    "description": "Enabled if Enterprise DC/OS is configured using the 'strict' security mode flag. When disabled, tasks run as 'root' by default.",
+                    "type": "boolean",
+                    "default": false
+                },
+                "strict-mode-user": {
+                    "description": "When strict mode is enabled in Enterprise DC/OS, tasks should run as the specified user. The 'strict' security mode does not permit running tasks as root. By default, the user 'nobody' is used.",
```

Comment:
Change first sentence to: `In strict mode, tasks will run as this user.`
Add: `Ensure that this user exists on each of the agents in your DC/OS cluster.`